src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -4383,6 +4383,18 @@ void emitter::emitInsSve_R_R_R(instruction     ins,
         case INS_sve_ld1w:
         case INS_sve_ld1sw:
         case INS_sve_ld1d:
+        case INS_sve_ld2b:
```

Comment:
you need to add asserts that the underlying registers are indeed  consecutive. For reference, see the implementation of `Sve_StoreAndZip*` in `hwintrinsiccodegenarm64.cpp`.

Reply:
Sure

Reply:
So, it can't be exactly like what `Sve_StoreAndZip` is doing because `Sve_StoreAndZip` checks it via by its op3:
```cpp
                assert(intrin.op3->OperIsFieldList());
                GenTreeFieldList* fieldList  = intrin.op3->AsFieldList();
                GenTree*          firstField = fieldList->Uses().GetHead()->GetNode();
                op3Reg                       = firstField->GetRegNum();

#ifdef DEBUG
                unsigned  regCount = 0;
                regNumber argReg   = op3Reg;
                for (GenTreeFieldList::Use& use : fieldList->Uses())
                {
                    regCount++;

                    GenTree* argNode = use.GetNode();
                    assert(argReg == argNode->GetRegNum());
                    argReg = getNextSIMDRegWithWraparound(argReg);
                }
```
For `Load*xVector`, it's the `node` and destination registers. I'll see if we can get this information.